packages/@react-spectrum/s2/src/Card.tsx

```diff
@@ -0,0 +1,669 @@
+  position: 'relative',
+  transition: 'default',
+  overflow: 'clip',
+  marginX: '[calc(var(--card-padding-x) * -1)]',
+  marginTop: '[calc(var(--card-padding-y) * -1)]',
+  marginBottom: {
+    ':last-child': '[calc(var(--card-padding-y) * -1)]'
+  },
+  borderRadius: {
+    isQuiet: borderRadius
+  },
+  boxShadow: {
+    isQuiet: {
+      isHovered: 'elevated',
+      isFocusVisible: 'elevated',
+      isSelected: 'elevated'
+    }
+  },
+  ...focusRing(),
+  outlineStyle: {
+    default: 'none',
+    isQuiet: {
+      isFocusVisible: 'solid'
+    }
+  }
+});
+
+const image = style({
+  width: 'full',
+  aspectRatio: '[3/2]',
+  objectFit: 'cover',
+  userSelect: 'none',
+  pointerEvents: 'none'
+});
+
+let title = style({
+  font: 'title',
+  fontSize: {
+    size: {
+      XS: 'title-xs',
+      S: 'title-xs',
+      M: 'title-sm',
+      L: 'title',
+      XL: 'title-lg'
+    }
+  },
+  lineClamp: 3
+});
+
+let description = style({
+  font: 'body',
+  fontSize: {
+    size: {
+      XS: 'body-2xs',
+      S: 'body-2xs',
+      M: 'body-xs',
+      L: 'body-sm',
+      XL: 'body'
+    }
+  },
+  lineClamp: 3,
+  gridColumnEnd: 'span 2'
+});
+
+let content = style({
+  display: 'grid',
+  // By default, all elements are displayed in a stack.
+  // If an action menu is present, place it next to the title.
+  gridTemplateColumns: {
+    ':has([data-slot=menu])': ['minmax(0, 1fr)', 'auto']
+  },
+  columnGap: 4,
+  flexDirection: 'column',
+  flexGrow: 1,
+  alignItems: 'baseline',
+  alignContent: 'space-between',
+  rowGap: {
+    default: 8,
+    size: {
+      XS: 4
+    }
+  },
+  paddingTop: {
+    default: '--card-spacing',
+    ':first-child': 0
+  },
+  paddingBottom: {
+    default: '[calc(var(--card-spacing) * 1.5 / 2)]',
+    ':last-child': 0
+  }
+});
+
+let footer = style({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'end',
+  justifyContent: 'space-between',
+  gap: 8,
+  paddingTop: '[calc(var(--card-spacing) * 1.5 / 2)]'
+});
+
+export const CardViewContext = createContext<'div' | typeof GridListItem>('div');
+export const CardContext = createContext<ContextValue<Partial<CardProps>, DOMRefValue<HTMLDivElement>>>(null);
+
+interface InternalCardContextValue {
+  isQuiet: boolean,
+  size: 'XS' | 'S' | 'M' | 'L' | 'XL',
+  isSelected: boolean,
+  isHovered: boolean,
+  isFocusVisible: boolean
+}
+
+const InternalCardContext = createContext<InternalCardContextValue>({
+  isQuiet: false,
+  size: 'M',
+  isSelected: false,
+  isHovered: false,
+  isFocusVisible: false
+});
+
+const actionButtonSize = {
+  XS: 'XS',
+  S: 'XS',
+  M: 'S',
+  L: 'M',
+  XL: 'L'
+} as const;
+
+export const Card = forwardRef(function Card(props: CardProps, ref: DOMRef<HTMLDivElement>) {
+  [props] = useSpectrumContextProps(props, ref, CardContext);
+  let domRef = useDOMRef(ref);
+  let {density = 'regular', size = 'M', variant = 'primary', UNSAFE_className = '', UNSAFE_style, styles} = props;
+  let isQuiet = variant === 'quiet';
```

Comment:
<img width="579" alt="Screenshot 2024-09-03 at 2 28 56 PM" src="https://github.com/user-attachments/assets/77db3c3f-d269-48e5-8760-cafd8cf6fbea">

Should the background be different for the quiet variant here, or are we using the wrong background in storybook?

Reply:
one of the non-supported combinations?

Reply:
hmm, yes, the background is wrong in storybook docs. I originally had `transparent` here but then realized I shouldn't override `--s2-container-bg` in that case (it is used for the border around avatars etc.). Guess I should add it back.